.github/workflows/release.yml

```diff
@@ -109,8 +113,13 @@ jobs:
         name: 'Prepare CDN release'
         run: echo "versions=$(npm run --silent ci:prepare-release)" >> ${GITHUB_OUTPUT}
 
+      - name: generate build provenance
```

Comment:
Wondering if this should also only trigger if `hashFiles('packages/rum/dist/bundles/*.js')` ?

Reply:
```suggestion
      - name: generate build provenance
      if: ${{ hashFiles('packages/rum/dist/bundles/*.js') }}
```